### DISCUSSION TITLE
How to create snippet for a blank file with no filetype?

### DISCUSSION THREAD

**Discussion by davidjuniorsaylor** [atom/atom#23709] [Q&A]
What would I put in the file extension attribute or w/e its called to make this work in my snippets.cson file? Or is there another way to go about this?

'?????????, .text.html, .text.html.php':
	'New HTML':
		'prefix': '<html'
		'body': """
			<html>

			<head>
			</head>

			<body>
			</body>

			</html>
			"""

**Comment by juancarlosso** [comment]
Did you try the global scope?    
'*'
